mycore-xeditor/src/main/resources/components/xeditor/config/mycore.properties

```diff
@@ -44,6 +44,9 @@ MCR.ContentTransformer.xeditor.Class=org.mycore.common.content.transformer.MCRXS
 MCR.ContentTransformer.xeditor.TransformerFactoryClass=org.apache.xalan.processor.TransformerFactoryImpl
 MCR.ContentTransformer.xeditor.Stylesheet=xsl/xeditor-templates.xsl,xsl/xeditor.xsl
 
+# Comma separated list of uri resolvers disallowed in the xed:source/@uri
+MCR.XEditor.MCRXEditorTransformer.URISchema.disallowList=callJava:
+
```

Comment:
with or without colon - I am not sure
If we keep the colon,  the value works like an URI prefix and we could define more complex URIs including server with server and pathes if needed
If we remove the colon, the value is a "proper" URI schema and we should at the colon in the validating Java code.